bazel/deps.bzl

```diff
@@ -11,19 +11,19 @@ def deps(repo_mapping = {}):
     maybe(
         git_repository,
         name = "boringssl",
-        commit = "b9232f9e27e5668bc0414879dcdedb2a59ea75f2",
+        commit = "652d66d1feb8ba612e776e03182fa1c8f716d265",
```

Comment:
Where did you get this commit? I don't see this commit anywhere in the history of https://boringssl.googlesource.com/boringssl at all?

(How you selected the commit would have been a good thing to put in the PR description)

Reply:
one of the latest versions of boringssl (they do not usually cut releases) from the `main-with-bazel branch`
https://github.com/google/boringssl/commits/main-with-bazel/

Reply:
Aaaah they have a separate `main-with-bazel` branch - cool, thanks. Let's maybe leave a little comment here saying that, for future updaters who don't know that either.